spec/models/skill_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Skill, type: :model do
```

Comment:
so, yes it is not good idea to run test 1000 times your server resources would died on you from the start with tests like this one.